### PR TITLE
Add list of supported tags to README.md

### LINKED ISSUE #19: Add list of available tags with links to their Dockerfiles in the README

### PR DESCRIPTION
**Pull request by eguiraud**
...plus minor text fixes.
This fixes #19.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+## Quick reference
+
+Maintained by: [the ROOT team](https://root.cern/) and collaborators
+
+Where to get help: [the ROOT forum](https://root-forum.cern.ch/)
+
+## Supported tags and respective Dockerfile links
+
+* [`latest`](https://github.com/root-project/root-docker/blob/6.20.06-fedora32/fedora/Dockerfile), [`6.20.06-fedora32`](https://github.com/root-project/root-docker/blob/6.20.06-fedora32/fedora/Dockerfile)
```
Comment (amadio) [inline README.md:9]: I think that having latest point to something not using 6.22 can be confusing. Maybe it's time to switch to a different image for this tag?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+* [`6.22.00-conda`](https://github.com/root-project/root-docker/blob/6.22.00-conda/conda/Dockerfile)
+* [`6.20.06-arch`](https://github.com/root-project/root-docker/blob/6.20.06-arch/conda/Dockerfile)
+* [`6.20.06-centos7`](https://github.com/root-project/root-docker/blob/6.20.06-centos7/cc7/Dockerfile)
+* [`6.22.00-ubuntu20.04`](https://github.com/root-project/root-docker/blob/6.22.00-ubuntu20.04/ubuntu/Dockerfile)
```
Comment (amadio) [inline README.md:13]: See also https://hub.docker.com/r/rootproject/root/dockerfile. I don't know if it's possible, but maybe we can make this point to the right file for each tag and link to that instead?

File: README.md (hunk 1 above)
Comment (eguiraud) [reply README.md:9]: it's temporary. if we make it point to something else, next release maybe _that_ will take more time to move to 6.22.

File: README.md (hunk 2 above)
Comment (eguiraud) [reply README.md:13]: as far as I can tell that page is per image, not per tag?

File: README.md (hunk 1 above)
Comment (eguiraud) [reply README.md:9]: please open an issue if you really think we should switch, merging this for now as this PR is orthogonal to what we use as `latest`

**Pull request merged**